db/test-data.sql

```diff
@@ -170,3 +171,4 @@ COPY public.report_project (id, report_id, project_uuid) FROM stdin;
 COPY public.user_project (id, user_id, project_uuid, editor) FROM stdin;
 1	1	f93ccc54-f701-4da8-b816-696e166f3274	t
 \.
+SELECT setval('public.user_project_id_seq', COALESCE((SELECT MAX(id) FROM public.user_project), 1));
```

Comment:
If we're doing this, should we remove the id column from the copy command? Don't know if this is easier to understand/manipulate, though.

Reply:
sure, doesn't matter either way, have to increment still.